test/memusage-test.js

```diff
@@ -42,6 +43,7 @@ describe('Leakage check', function() {
                 let idx = 0;
                 lp.on('log', function(msg) {
                     let info = glp.glpMemInfo()
+                    console.log(`INDEX: ${idx++}: ${msg.trim()}`)
```

Comment:
@jbeverly cannot comment below, but I logged the info object and it is all zeros. The test below will pass if everything is zero, so we should probably update. See:
```
INDEX: 22: +     3: mip =   7.333333333e+02 <=     tree is empty   0.0% (0; 1)
{
  "count": 0,
  "cpeak": 0,
  "total": 0,
  "tpeak": 0
}
INDEX: 23: INTEGER OPTIMAL SOLUTION FOUND
{
  "count": 0,
  "cpeak": 0,
  "total": 0,
  "tpeak": 0
}
```

Reply:
actually, I see a different behavior when logging in `problem-memusage-tests.js`